packages/eas-cli/src/analytics.ts

```diff
@@ -32,11 +34,33 @@ export async function initAsync(): Promise<void> {
   }
   const amplitudeEnabled = await UserSettings.getAsync('amplitudeEnabled', true);
   if (amplitudeEnabled) {
-    const apiKey =
+    const analyticsConfig =
       process.env.EXPO_STAGING || process.env.EXPO_LOCAL
```

Comment:
When testing locally, if `EXPO_LOCAL` or `EXPO_STAGING` is enabled the build command does not succeed. The CLI asks me to log in, then it throws an invalid username/pass error. Does this env variable cause all authenticated commands to fail? is this intentional?

Reply:
Setting `EXPO_LOCAL` or `EXPO_STAGING` makes EAS CLI use another API endpoint than the production one.
If you want to use the local API, you need to have the local www running on your machine. Staging should work out of the box. 
Maybe you got the invalid username/password error because you used your production credentials? Staging doesn't use the same database as production, so if you want to use staging you should register another account at staging.expo.dev

Reply:
gotcha! thank you for the explanation. yeah, everything seems good to me then.
> Maybe you got the invalid username/password error because you used your production credentials
yup, this is definitely what happened